gitlab/http_backends/requests_backend.py

```diff
@@ -10,3 +11,40 @@ def __init__(self, session: Optional[requests.Session] = None) -> None:
     @property
     def client(self) -> requests.Session:
         return self._client
+
+    def http_request(
+        self,
+        method: Union[str, bytes],
```

Comment:
This can only ever be a string, right? It can only really be one of `get`,`post`,`put`,`delete` etc.

```suggestion
        method: str,
```